web/packages/shared/components/Notification/Notification.tsx

```diff
@@ -241,4 +241,6 @@ const Container = styled(Flex)`
   color: ${props => props.theme.colors.text.main};
   border-radius: 4px;
   cursor: pointer;
+  // Break up long addresses.
+  word-break: break-word;
```

Comment:
Without this, we'd cut off longer addresses in a weird way (due to `text-overflow: ellipsis`), while still allowing for text below them.

| Before | After |
| --- | --- |
| ![word-break-before](https://github.com/gravitational/teleport/assets/27113/d1f1f5dd-6939-4cb9-99e0-644c2fde180c) | ![word-break-after](https://github.com/gravitational/teleport/assets/27113/8b5eab8c-d63d-4b1c-a047-9faf319bb4c9) |